sirepo/template/raydata.py

```diff
@@ -38,10 +38,16 @@ def stateless_compute_scan_fields(data):
 
 
 def _request_scan_monitor(data):
+    c = sirepo.feature_config.for_sim_type(SIM_TYPE)
     try:
         r = requests.post(
-            sirepo.feature_config.for_sim_type(SIM_TYPE).scan_monitor_url,
+            c.scan_monitor_url,
             json=data,
+            headers=PKDict(
+                {
+                    sirepo.util.AUTH_HEADER: f"{sirepo.util.AUTH_HEADER_SCHEME_BEARER} {c.scan_monitor_api_secret}"
```

Comment:
Why is it in sirepo.util?

Reply:
It is shared with sim_db_file so it needs to be somewhere both can access. Is there somewhere else you think would be better?

Reply:
sirepo.const might be better since it is a constant, not a utility. Also, you might say HTTP_AUTH_SCHEME_BEARER. It's not technically a "header", rather a value. A bit arbitrary, and not important.

Reply:
Ok, ended up making a function for it in sirepo.util and moving the const's to sirepo.const 2f0e408c9

Reply:
The "POSIT" bothers me. These are all coupled so they should be in one place. I didn't realize the generation was going on. Seems like unique_key and the rest of it should be in util. It's a bit arbitrary, but the context matters.